### PR TITLE
Also test for src:alt_label, ditch parent_id test

### PR DESCRIPTION
**Pull request by nvkelso**
Connects with https://github.com/whosonfirst/go-whosonfirst-geojson-v2/issues/6.

Not tested as @stepps00 still needs to make the data changes.

### CODE REVIEW

**Comment by stepps00** [comment]
I was able to build a combined distribution file using all 260 admin repos with correct feature counts with these changes. Next step is to test this using the `-index-alt-files` flag.

Here is the output of running the dist code: https://gist.github.com/stepps00/20094974ea67f6660e4f4f002d6f388c

@thisisaaronland - can we merge this change to get us a step closer to updating the public "combined" dist file on dist.whosonfirst.org?

File: properties/whosonfirst/whosonfirst.go [hunk 1]
```diff
@@ -279,21 +279,27 @@ func IsAlt(f geojson.Feature) bool {
...
 
 	// we used to test that wof:parent_id wasn't -1 but that's a bad test since
 	// plenty of stuff might have a parent ID of -1 and really what we want to
 	// test is the presence of the property not the value
 	// (20190821/thisisaaronland)
 
-	if !utils.HasProperty(f.Bytes(), []string{"properties.wof:parent_id"}) {
-		return true
-	}
-
```
Comment (thisisaaronland) [inline properties/whosonfirst/whosonfirst.go:296]: I am not sure I understand the point of removing this test. Can you elaborate?

Comment (nvkelso) [reply properties/whosonfirst/whosonfirst.go:296]: There are more false results than valid results with this test. There are plenty of WOF records (including neighbourhoods that are not in a locality; data that is on the coast but just outside of country polygon) that have `parent_id = -1`, and for some reason they are being returned with this logic. 

It looks like the test originally was to target just alt geometries, and now that we have converted to explicate data properties, this generic and problematic test isn't needed anymore.

Comment (straup) [reply properties/whosonfirst/whosonfirst.go:296]: The test is to ensure that a feature _has_ a `wof:parent_id` property which has always been required, regardless of its value.

If you're saying that a record with a value of `-1` is triggering this error that's a logic bug in the code. If that's the case do you have an example ID I can test against.

Comment (stepps00) [reply properties/whosonfirst/whosonfirst.go:296]: example where a `-1` parent_id was causing an is_alt flag of `1` to be set (but there are not alt files): 890444831

example where a `-1` parent_id was causing an is_alt flag of `1` to be set (and the record has alt files): 85850447

Comment (nvkelso) [reply properties/whosonfirst/whosonfirst.go:296]: The 2nd example is common for 10s of thousands of records, like neighbourhoods that had funky Quattroshape geometries that were reset to points for their default geoms, but the QS shape was kept as an alt.

Comment (thisisaaronland) [reply properties/whosonfirst/whosonfirst.go:296]: Oh I see. Sorry, the confusion was my bad. I managed (for all this time) to think you were removing the general `wof:parent_id` check not the one is `IsAlt`. Tasteful muzak plays...

**Pull request merged**